META.github.template

```diff
@@ -0,0 +1,11 @@
+# JBUILDER_GEN
+
+package "js" (
+  description = "Deprecated. Use github-js directly"
+  requires = "cohttp-js"
+)
+
+package "unix" (
+  description = "Deprecated. Use github-unix directly"
+  requires = "github-unix"
+)
```

Comment:
it's useful to keep this for a release so that old packages can just have their opam build deps changed, rather than having to re-release or upper-bound them.